Source/Core/Core/HW/VideoInterface.cpp

```diff
@@ -107,7 +107,7 @@ void Preset(bool _bNTSC)
   state.vertical_timing_register.EQU = 6;
   state.vertical_timing_register.ACV = 0;
 
-  state.display_control_register.ENB = 1;
+  state.display_control_register.ENB = _bNTSC ? 0 : 1;
```

Comment:
How did you arrive at this? According to [YAGCD](http://hitmen.c02.at/files/yagcd/yagcd/chap5.html#sec5.3), `ENB` is 1 in all cases (even for NTSC)...which doesn't necessarily mean that it is _correct_ like that.

Reply:
It was default from 4.0-7542 to 4.0-7592 and only reverted because of issues with PAL titles. I don't know that accurate since I've done no hardware tests to confirm